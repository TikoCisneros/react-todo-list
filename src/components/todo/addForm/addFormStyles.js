import styled from "styled-components";

const FormInput = styled.input`
  margin: 0 1rem 20px;
  font-size: 20px;
  line-height: 1.4em;
  color: #2979FF;
  padding: 8px 6px;
  border: none;
  box-shadow: 0 2px 1px -2px #757575;
  ::placeholder {
    color: #cdcdcd;
    font-style: italic;
    font-size: 20px;
  }
  :focus {
    outline: 0;
  }
`;

export { FormInput };
