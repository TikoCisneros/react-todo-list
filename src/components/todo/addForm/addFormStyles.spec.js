import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { FormInput } from './addFormStyles';

describe('Add Form components', () => {
  it('FormInput render successfully', () => {
    const tree = renderer.create(<FormInput />).toJSON()
    expect(tree).toMatchSnapshot()
  });
});
