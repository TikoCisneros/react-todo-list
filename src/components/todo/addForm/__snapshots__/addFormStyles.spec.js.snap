// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Form components FormInput render successfully 1`] = `
.c0 {
  margin: 0 1rem 20px;
  font-size: 20px;
  line-height: 1.4em;
  color: #2979FF;
  padding: 8px 6px;
  border: none;
  box-shadow: 0 2px 1px -2px #757575;
}

.c0::-webkit-input-placeholder {
  color: #cdcdcd;
  font-style: italic;
  font-size: 20px;
}

.c0::-moz-placeholder {
  color: #cdcdcd;
  font-style: italic;
  font-size: 20px;
}

.c0:-ms-input-placeholder {
  color: #cdcdcd;
  font-style: italic;
  font-size: 20px;
}

.c0::placeholder {
  color: #cdcdcd;
  font-style: italic;
  font-size: 20px;
}

.c0:focus {
  outline: 0;
}

<input
  className="c0"
/>
`;
